<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>956</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
      <item>
       <widget class="QCustomPlot" name="customPlot" native="true">
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>956</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoadDataFile"/>
    <addaction name="actionReloadDataFile"/>
    <addaction name="separator"/>
    <addaction name="actionExportImage"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuGraph">
    <property name="title">
     <string>Graph</string>
    </property>
    <widget class="QMenu" name="menuShowHide">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Show/Hide</string>
     </property>
    </widget>
    <addaction name="actionAutoScaleXAxis"/>
    <addaction name="actionAutoScaleYAxis"/>
    <addaction name="separator"/>
    <addaction name="actionSetManualScaleXAxis"/>
    <addaction name="actionSetManualScaleYAxis"/>
    <addaction name="separator"/>
    <addaction name="menuShowHide"/>
    <addaction name="separator"/>
    <addaction name="actionWatchFile"/>
    <addaction name="actionDynamicSession"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuGraph"/>
   <addaction name="menu"/>
  </widget>
  <action name="actionLoadDataFile">
   <property name="text">
    <string>Load Data File...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionReloadDataFile">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Reload Data File</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="actionExportImage">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Export Image</string>
   </property>
  </action>
  <action name="actionAutoScaleXAxis">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Auto scale x-axis</string>
   </property>
  </action>
  <action name="actionAutoScaleYAxis">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Auto scale y-axis</string>
   </property>
  </action>
  <action name="actionSetManualScaleXAxis">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Set Manual scale x-axis...</string>
   </property>
  </action>
  <action name="actionSetManualScaleYAxis">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Set Manual scale y-axis...</string>
   </property>
  </action>
  <action name="actionMeu">
   <property name="text">
    <string>Meu</string>
   </property>
  </action>
  <action name="actionWatchFile">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Watch File</string>
   </property>
  </action>
  <action name="actionDynamicSession">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Dynamic Session</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
